tsconfig.json

```diff
@@ -19,9 +19,9 @@
     ],
     "lib": [
       "dom",
-      "es5",
       "scripthost",
-      "es2015.collection"
+      "es2015",
+      "es2019.array"
```

Comment:
One more thing, do we have autopolyfilling for latest browsers > 0.5%? Should I worry about using `Array.prototype.flatMap`, @sonic16x ?